asvdb/asvdb.py

```diff
@@ -151,18 +154,27 @@ def __init__(self, dbDir,
         self.projectName = projectName
         self.commitUrl = commitUrl
 
+        self.machineFileExt = path.join(self.defaultResultsDirName, "*", self.machineFileName)
```

Comment:
I'm used to the suffix `Ext` meaning "extension", but it looks like these are used for paths and file names. Does `Ext` mean something for S3?

Reply:
It is the file path extension to that specific file/folder from the base asv folder. If your results are being stored in `/tmp/asv` then the extension for getting to the `benchmarks.json` is `/results/benchmarks.json`. In AS3 there is the `bucket`, and the `key` (basically a filepath), and I needed a way to separate out the bucket itself, the base asv path, and the extra extension needed to access specific files from that base.